app/assets/stylesheets/src/ats/_positions.scss

```diff
@@ -26,84 +26,90 @@
 
   .pipeline-column {
     min-width: 252px;
-  }
-
-  .pipeline-stage-header {
-    height: $pipeline-header-height;
-  }
 
-  .card {
-    position: relative;
-    .card-avatar {
-      width: 64px;
-      height: 64px;
-      .avatar {
-        --tblr-avatar-icon-size: 2.5rem;
-      }
-      img {
-        width: 100%;
-        height: auto;
-      }
+    @include media-breakpoint-up(sm) {
+      background-color: $card-bg;
+      border: $card-border-width solid $card-border-color;
```

Comment:
https://github.com/freeats/freeats/issues/72#issuecomment-2458805335

The standard color of the card borders `rgba(4, 32, 69, 0.1)` is used here